cache/sampleStore.js

```diff
@@ -38,6 +38,11 @@ const constants = {
   objectType: { aspect: 'aspect', sample: 'sample', subject: 'subject' },
   prefix: PFX,
   separator: SEP,
+  commands: {
+    hgetall: 'hgetall',
+    hget: 'hget',
+    smembers: 'smembers',
+  },
 };
```

Comment:
I guess it should be fine. I am mostly worried about typos. http://rc3.org/2006/10/11/use-contants-instead-of-string-literals/